scripts/src/main/resources/scripts/command/aws

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env bash
+
+# autocompletion list
+if [ "${1}" = "shortlist" ]
+then
+  if [ -z "${2}" ]
+  then
+    echo "setup help"
+  fi
+  exit
+fi
+
+if [ -n "${DEVON_IDE_TRACE}" ]; then set -vx; fi
+# shellcheck source=scripts/functions
+source "$(dirname "${0}")"/../functions
+
+# Call aws with specified arguments.
+function doRun() {
+  doSetup silent
+  doRunCommand "${AWS_HOME}/aws" "${@}"
+}
+
+# $1: path_to_package
+# $2: enable_ide_update
+function doPackageInstall() {
+  local path_to_package="${1}"
+  local enable_ide_update="${2}"
+  local version_file="${path_to_package}/.devon.software.version"
+  local current_version
+  current_version="$(cat "${version_file}")"
+
+  if doIsWindows
+  then
+    doEcho "Windows Installation..."
+    windows_path_to_package=$(cygpath -w "${path_to_package}")
+    powershell.exe -Command "Start-Process msiexec.exe -verb runas -Wait -ArgumentList '/I ${windows_path_to_package}\aws-${current_version}-AWSCLIV2.msi /quiet'"
+  elif doIsMacOs
+  then
+    doEcho "Installating aws for MacOS..."
+  else
+    doEcho "Installating aws for Linux..."
+    doRunCommand "sudo ${path_to_package}/install"
+  fi
+  if [ "${enable_ide_update}" == "y" ]
+  then
+    if [ ! -d "${DEVON_IDE_HOME}/software/aws/" ]
+    then
+      doRunCommand "mkdir ${DEVON_IDE_HOME}/software/aws/"
+    fi
+    doRunCommand "cp ${version_file} ${DEVON_IDE_HOME}/software/aws/"
+  fi
+  #doRunCommand "rm -rf ${path_to_package}"
+}
+
+function doSetup() {
+
+  # start installation after extract
+  local inst_pack="y"
+  local enable_ide_update="n"
+
+  # Check if AWS_HOME is already set
+  if [ -z "${AWS_HOME}" ]
+  then
+    if [ "${inst_pack}" == "y" ]
+    then
+      AWS_HOME="${DEVON_IDE_HOME}/updates/install/aws"
+    else
+      AWS_HOME="${DEVON_IDE_HOME}/software/aws"
+    fi
+  fi
+  if [ ! -d "${AWS_HOME}" ] || [ "${1}" != "silent" ]
+  then
+    # Get leatest release
+    if [ -z "${AWS_VERSION}" ]
+    then
+      doEcho "Getting latest release..."
+      AWS_VERSION=$(curl "https://github.com/aws/aws-cli/tags" | awk -F'tags/' '/archive/ { print $2}' | sort -r | head -1 | awk -F'.zip' '{print $1}')
+    fi
+
+    doInstall "-" "${AWS_HOME}" "aws" "${AWS_VERSION}" 
+    if [ "${?}" = 0 ]
+    then
+      if [ "${inst_pack}" == "y" ]
+      then
+        doPackageInstall "${AWS_HOME}" "${enable_ide_update}"
+      fi
+    fi
+  fi
+
+  if [ "${1}" != "silent" ] && ! doIsQuiet
+  then
+    doEcho "logon again to test aws with \"aws version\""
+  fi 
+
+}
+
+# CLI
+case ${1} in 
+"help" | "-h")
+  echo "Install aws."
+  echo
+  echo "Arguments:"
+  echo " setup                    install aws on your machine."
+  echo " <<args>>                 call aws with the specified arguments. Call aws --help for details or use helm directly as preferred."
+  echo
+;;
+"setup" | "s" | "")
+  doEcho "Installating aws..."
```

Comment:
The doSetup command should already do the logging and honor the `silent` option, so simply remove this line.

Reply:
I corrected it